src/components/wizard/wizard.component.ts

```diff
@@ -347,12 +351,37 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
         return this.steps.toArray()[index];
     }
 
+    /**
+     * If a step in the wizard becomes invalid, all steps sequentially after
+     * it, should become unvisited and incomplete
+     */
+    protected setNextStepsUnvisited(): void {
+
+        if (!this.resetVisitedOnValidationError) {
+            return;
+        }
+
+        this.getFutureSteps().forEach(step => {
+            // if the step is not the current step then also mark it as unvisited
+            if (step.visited && step !== this.getCurrentStep()) {
+                step.setVisited(false);
+            }
+        });
+    }
+
+    /**
+     * Get the steps ahead of the currently active steps
```

Comment:
This is an incorrect description and function name, since it includes the currently active step.

Reply:
Apologies this should have been updated to match the functionality.